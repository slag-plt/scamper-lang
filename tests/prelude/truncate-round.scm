(truncate 0.71)
(truncate 111)
(truncate 6.1)
(truncate 0.69)
(truncate 0.10000000000000009)
(truncate 0)
(round 0.71)
(round 111)
(round 6.1)
(round 0.69)
(round 0.10000000000000009)
(round 0)