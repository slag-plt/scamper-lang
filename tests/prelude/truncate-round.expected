0
111
6
0
0
0
1
111
6
1
0
0
